helm-org-ql.el

```diff
@@ -178,21 +178,27 @@ Is transformed into this query:
   "Return Helm source named NAME that searches BUFFERS-FILES with `helm-org-ql'."
   ;; Expansion of `helm-build-sync-source' macro.
   (helm-make-source name 'helm-source-sync
+    :filtered-candidate-transformer (lambda (candidates _)
```

Comment:
Looking at the docstring for `helm-source`, it might be best to use `filter-one-by-one`.

Reply:
I tried to use it, but failed - I get the only candidate "Org Agenda files", which is odd. Probably, I misunderstand the docstring of that transformer.

Reply:
It seems that `filter-one-by-one` does not work with the current source type. It is generally used in buffer sources where contents of the buffer is changing. The `filter-one-by-one` transformed in never run in sync source.